src/Compiler/AbstractIL/ilread.fs

```diff
@@ -878,10 +878,10 @@ let hfmCompare (TaggedIndex(t1: HasFieldMarshalTag, idx1)) (TaggedIndex(t2: HasF
     elif idx1 > idx2 then 1
     else compare t1.Tag t2.Tag
 
-let tomdCompare (TaggedIndex(t1: TypeOrMethodDefTag, idx1)) (TaggedIndex(t2: TypeOrMethodDefTag, idx2)) =
-    if idx1 < idx2 then -1
-    elif idx1 > idx2 then 1
-    else compare t1.Tag t2.Tag
+let inline tomdCompare (t1: TaggedIndex<TypeOrMethodDefTag>) (t2: TaggedIndex<TypeOrMethodDefTag>) =
+    if t1.index < t2.index then -1
+    elif t1.index > t2.index then 1
+    else compare t1.tag t2.tag
```

Comment:
Why not destructuring as before? This adds additional field/property access. Whereas before index was bound explicitly.

Reply:
This gave some small perf improvement. But ok, removed from this PR

Reply:
I was mostly curious why as it just looked as if it was adding field access vs binding directly